spec/features/merchant_items/new_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'create merchant item page' do
+  before :each do
+    @merchant1 = create(:merchant)
+  end
+
+  it 'can link from the merchant index page to the new merchant item page' do
+    visit merchant_items_path(@merchant1)
+
+    click_link('Create a New Item')
+
+    expect(current_path).to eq(new_merchant_item_path(@merchant1))
+  end
+
+  it 'can create a new merchant item' do
+    visit new_merchant_item_path(@merchant1)
+
+    expect(page).to have_content('Create a New Merchant Item')
+
+    fill_in('Name', with: 'Diagon Alley')
+    fill_in('Description', with: "The truth. It is a beautiful and terrible thing, and should therefore be treated with great caution.")
+    fill_in('Unit Price', with: 48_730)
+
+    click_on('Submit')
+
+    # Then I am taken back to the items index page ** This means the merch item page, right?
+    expect(Item.last.name).to eq('Diagon Alley')
+    expect(Item.last.description).to eq('The truth. It is a beautiful and terrible thing, and should therefore be treated with great caution.')
+    expect(Item.last.unit_price).to eq(48_730)
+    expect(Item.last.enabled).to eq(false)
```

Comment:
I noticed that the default value for `enabled` in the schema is `true`.  I wonder if we should update the default for `enabled` to be `false` with a migration ... or just send it through the form like you have it now.  I had gone the migration route, but maybe we can talk and see which way makes more sense.

Reply:
Happy to go over the pros and cons of each.